test/controllers/update/overseas.entity.presenter.controller.spec.ts

```diff
@@ -90,6 +92,13 @@ describe("OVERSEAS ENTITY PRESENTER controller", () => {
       expect(resp.status).toEqual(500);
       expect(resp.text).toContain(SERVICE_UNAVAILABLE);
     });
+
+    test(`renders the ${UPDATE_PRESENTER_PAGE} page for remove`, async () => {
+      mockGetApplicationData.mockReturnValueOnce({ });
+      const resp = await request(app).get(`${OVERSEAS_ENTITY_PRESENTER_URL}${JOURNEY_REMOVE_QUERY_PARAM}`);
+
+      expect(resp.status).toEqual(302);
```

Comment:
you should test that the page is rendered correctly, the check the resp.text contains correct page title, remove banner, and the back link

Reply:
page has been tested and it renders correctly